### ISSUE TITLE
No way to keep changing denylist components permanently enabled #1

### ISSUE THREAD

**Issue by rabaimorp** [topjohnwu/Magisk#8568]
I know this is not a bug report, so feel free to throw it out, but I think this really should be implemented.

So nowadays more advanced root detection components keep changing every reboot so even if I add the app with all components to the denylist, I have to go and re-enable them every reboot because the names keep changing. An option to lock an app to automatically add all its components to the denylist would be a perfect fix for this issue. Also I can't imagine it beeing to hard to implement.

Device: Samsung Galaxy A53 5G
Android version: 14
Magisk version name: 28
Magisk version code: 28001

**Comment by canyie** [comment]
Hello, your issue seems to be caused by app update. I know an app can sliently update itself  on Android 12+, and turning off the "install unknown source app" permission can prevent this. So maybe you can try it. Disabling auto app update in app store settings may also help.

**Comment by rabaimorp** [comment]
No, I've did multiple reboots in succession and the components changed almost every time. There is no way that the app had a rapid-fire update streak just when I did that, not later, not ealrier.

**But even if this was the case a way to lock all of an apps components to the denylist, would still be ideal**

**Comment by vvb2060** [comment]
Please report to Google, this is a serious Android security vulnerability and needs to be fixed by AOSP.

**Comment by canyie** [comment]
Which app are you using?

**Comment by heinrich26** [comment]
I've observed a similar behavior with the VR SecureGo plus app (which is like a fancy OTP app by my bank). I was previously thinking they were rotating their isolated subproc names (or whatever they're called) on boot, but that doesn't seem to be the case. (the names are just gibberish: 

<img width="960" height="598" alt="Image" src="https://github.com/user-attachments/assets/3739f7af-fe2c-4e28-8822-7588f5e2d82f" />

Anyways, every single time I come to use the app, the isolated procs are gone from my denylist. They're probably just updating their app like every second day, which is annoying nonetheless.

I've tried writing a module to add it to the denylist automatically, but my boot script doesn't seem to work as part of a module...
```sh
pm dump de.fiduciagad.securego.vr | grep process=de.fiduciagad.securego.vr: | awk -F '=' '{ split($2, a, " "); print a[1] }' | sort -u | xargs -n 1 magisk --denylist add isolated
```
When I tried using `service.sh`, it ran into a conflict with the database; with `post-fs-data.sh` nothing happens either... :( Do I need to add a `su` before?
